pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -817,13 +817,15 @@
         "1.1.37",
         "1.2.0",
         "1.1.36",
-        "1.3.0"
+        "1.3.0",
+        "1.3.1"
       ],
-      "BaselineVersion": "1.3.0",
+      "BaselineVersion": "1.3.1",
```

Comment:
Did you want to do this?  That's effectively causing a patch family dependency between any package that references this.